x-pack/typings/elastic__node_crypto.d.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+declare module '@elastic/node-crypto';
+declare module 'node-jose' {
```

Comment:
why it's a part of `x-pack`? can `request-crypto` provides all typings that it uses?

Reply:
`node-crypto` is also used by encrypted_saved_objects but is using `@ts-ignore`. `request-crypto` does provide its own typing but for some reason kibana fails because `node-jose` and `@elastic/node-crypto` types are not supplied. I'll check how to approach this differently.

Reply:
@restrry I have submitted a PR to add types for `@elastic/node-crypto` here https://github.com/elastic/node-crypto/pull/9

For the `node-jose` I supply the types now from `@types/node-jose` from the `request-crypto` package itself. So it is removed from this file.

Since this is a development only file we can update it anytime. I don't think it should block the PR from getting merged. I'll remove this file from the master branch once the PR is merged. What do you think?